etc/security/mh_default_org.xml

```diff
@@ -162,6 +162,8 @@
     <!-- Securing the URLs for the external API interface -->
     <!-- External API GET Endpoints -->
     <sec:intercept-url pattern="/api" method="GET" access="ROLE_ADMIN, ROLE_API"/>
+    <sec:intercept-url pattern="/api/agents" method="GET" access="ROLE_ADMIN, ROLE_API_CAPTURE_AGENTS_VIEW"/>
+    <sec:intercept-url pattern="/api/agents/*" method="GET" access="ROLE_ADMIN, ROLE_API, ROLE_API_CAPTURE_AGENTS_VIEW"/>
```

Comment:
@gregorydlogan But here, ROLE_API is not required. @krinnewitz will update the PR in a few moments.

Reply:
Done ;)